conf/messages.cy

```diff
@@ -564,6 +564,11 @@ goodsRecordSearchResult.title.withPage = Mae yna {0} o ganlyniadau chwilio – t
 goodsRecordSearchResult.h1 = Mae yna {0} o ganlyniadau chwilio
 goodsRecordSearchResult.p1 = Roeddech yn chwilio am y term {0}.
 
+goodsRecordSearchResult.p2 = {0} records found
```

Comment:
Might be worth adding a TODO above this, just so it's easier to spot when we come to put the welsh in when we receive it